crates/connectors/ndc-postgres/src/schema/mutation/v2.rs

```diff
@@ -117,25 +118,28 @@ pub fn update_to_procedure(
 
     // Make an object type for each column's update object.
     for (column_name, column_info) in &update_by_key.table_columns {
-        let (object_name, object_type) = make_update_column_type(
-            &update_by_key.collection_name,
-            column_name,
-            column_to_type(column_info),
-        );
-        // add to object types
-        object_types.insert(object_name.clone(), object_type.clone());
-        // Remember for the update_columns type
-        fields.insert(
-            column_name.clone(),
-            models::ObjectField {
-                description: Some(format!(
-                    "Update the '{column_name}' column in the '{}' collection.",
-                    update_by_key.collection_name
-                )),
-                r#type: models::Type::Named { name: object_name },
-                arguments: BTreeMap::new(),
-            },
-        );
+        // Add the column if it is not generated.
+        if let Some((object_name, object_type)) =
+            make_update_column_type(&update_by_key.collection_name, column_name, column_info)
+        {
+            // add to object types
+            object_types.insert(object_name.clone(), object_type.clone());
+            // Remember for the update_columns type
+            fields.insert(
+                column_name.clone(),
+                models::ObjectField {
+                    description: Some(format!(
+                        "Update the '{column_name}' column in the '{}' collection.",
+                        update_by_key.collection_name
+                    )),
+                    // We can not specify these.
+                    r#type: models::Type::Nullable {
```

Comment:
We don't have to specify any update we don't want to, so wrapping all of these in nullable.